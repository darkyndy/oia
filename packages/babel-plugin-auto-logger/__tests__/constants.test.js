// testing file
import pluginConstants from '../src/constants';

describe('constants.js', () => {
  it('is an object with plugin constants', () => {
    expect(pluginConstants).toMatchSnapshot();
  });
});
