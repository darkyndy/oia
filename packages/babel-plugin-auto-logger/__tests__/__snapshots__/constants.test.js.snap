// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants.js is an object with plugin constants 1`] = `
Object {
  "LOGGER_API": "console",
  "MEMBER_EXPRESSION_CATCH": "memberExpressionCatch",
}
`;
